src/components/app-content-renderer/recommendation-tile.js

```diff
@@ -67,28 +63,30 @@ const RecommendationGroup = ({
         <FlexItem>
           <GroupIcon
             className={classnames({
-              error: state === 'error',
-              warning: state === 'warning',
-              info: state === 'info',
-              green: state === 'success',
+              error: recommendation.state === 'error',
+              warning: recommendation.state === 'warning',
+              info: recommendation.state === 'info',
+              green: recommendation.state === 'success',
             })}
           />
         </FlexItem>
         <FlexItem>
           <TextContent>
-            {title && <Text component="h5">{title}</Text>}
-            <Text>{description}</Text>
+            {recommendation.title && (
+              <Text component="h5">{text(recommendation.title)}</Text>
+            )}
+            <Text>{text(recommendation.description)}</Text>
           </TextContent>
         </FlexItem>
       </Flex>
       <Button
         component="a"
         className="recommendation-button"
-        href={action.href}
+        href={recommendation.action.href}
```

Comment:
we should add an empty object as the action default. Or an error boundary that will unregister the field so we won't crash the whole landing page.

Reply:
It's required prop - all recommendation tiles have to have an action. I think that is sufficient.